src/pages/chanyoung/Login/components/Id.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+class Id extends React.Component {
+  checkVaild = ({ target: { value } }) => {
+    value.includes('@')
+      ? this.props.changeIsVaild('id', value, true)
+      : this.props.changeIsVaild('id', value, false);
+  };
```

Comment:
```suggestion
  checkVaild = (e) => {
    this.props.changeIsVaild('id', e.target.value)
  };
```
- value.includes('@')가 이미 boolean 입니다
- 코드를 정리하다 보면 2번 째 인자와 3번째 인자가 모두 value 값에 의존하고 있는데, 이미 value 전체를 넘겨주고 있으므로 T/F 판별은 changeIsValid 내에서 처리하면 될거 같습니다
- 그리고 구조분해 할당은 1 depth 정도가 충분한거 같아요

Reply:
완료했습니다